### DISCUSSION TITLE
Maintaining schema in two places?

### DISCUSSION THREAD

**Discussion by kaihendry** [sqlc-dev/sqlc#2209] [Issue Triage] [answered]
I don't understand your docs here: https://docs.sqlc.dev/en/stable/howto/ddl.html

I'm maintaining my database schema in two places and that doesn't seem right:

1. https://github.com/kaihendry/sqlc-tutorial/blob/main/migrations/20230417134833_init.sql
2. https://github.com/kaihendry/sqlc-tutorial/blob/main/schema.sql

What am I missing please?

I think my question is related to https://github.com/kyleconroy/sqlc/discussions/1928

**Comment by coolaj86** [comment]
@kaihendry I don't understand the question.

What is the purpose of `schema.sql`?

I don't have that in my projects and I don't know what it's for (or where it's referenced in the documentation).

I think you're just supposed to pick one - I think goose is what I would pick if I were starting over and hadn't already been using sql migrate - and you're done.

`sqlc` is _aware_ of those popular migration tools, but doesn't really do anything with them other than follow the convention of the folder name and the lexicographical sort of the files.

**Reply by kaihendry** [reply]
`schema.sql` drops out after doing the tutorial here: https://docs.sqlc.dev/en/latest/tutorials/getting-started-postgresql.html

**Reply by coolaj86** [reply]
I think that's just the "Hello World" migrator - no goose, etc.

**Comment by kaihendry** [comment, marked as answer]
I grokked what I need to do, just use the migrations directory: https://github.com/kaihendry/sqlc-tutorial/blob/main/sqlc.yaml#L5 and sqlc compiles it..

Thank you @coolaj86 !
